src/app/visit/visit.component.ts

```diff
@@ -113,4 +126,24 @@ export class VisitComponent implements OnInit {
     ): object is Patient {
         return Object.hasOwn(object, 'name');
     }
+
+    createVisitPointsTable(): void {
+        this.dataSource = new MatTableDataSource<Store>(
+            this.store.filter((s: Store) => this.visitPoints.includes(s.id)),
+        );
+        this.paginator.firstPage();
+        this.showFinish = true;
+        this.showCheck = false;
+        this.dataSource.paginator = this.paginator;
+    }
+
+    pageTriggerManually(): void {
+        if (!this.paginator.hasNextPage() && this.visitPoints.length >= 1) {
+            this.showCheck = true;
+        }
+    }
+
+    printNote(): void {
+        console.log(this.noteVal, this.dataSource.data);
+    }
```

Comment:
I would like to keep the private functions always at the bottom, first all public and then all private, please move the private function to the bottom of the file